package.json

```diff
@@ -49,7 +49,7 @@
   },
   "dependencies": {
     "bluebird": "^3.3.3",
-    "contentful-batch-libs": "^4.9.2",
+    "contentful-batch-libs": "beta",
```

Comment:
having the full version here is much better, once we remove the beta tag no changes will be required